gocourse05/zooobservation/main_test.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"testing"
+	"time"
+)
+
+func TestSaveToServer(t *testing.T) {
+	defer func() {
+		if r := recover(); r != nil {
+			t.Errorf("The code have panic")
+		}
+	}()
+
+	err := error(nil)
```

Comment:
Це те саме, але більш ідіоматично:
```suggestion
	var err error
```